website/addons/box/__init__.py

```diff
@@ -24,7 +24,7 @@
 ADDED_MANDATORY = []
 
 VIEWS = []
-CONFIGS = ['user', 'node']
+CONFIGS = ['accounts', 'node']
```

Comment:
Why was this changed? What is the difference between `users` and `accounts` configs?